bumble/keys.py

```diff
@@ -307,3 +307,30 @@ async def get(self, name: str) -> Optional[PairingKeys]:
             return None
 
         return PairingKeys.from_dict(keys)
+
+
+# -----------------------------------------------------------------------------
+class InMemoryStore(KeyStore):
```

Comment:
suggest: naming this class just `MemoryKeyStore` (having the word `Key` in there makes it explicit what this stores, and the `In` prefix isn't really needed. This also parallels the name of `JsonKeyStore`.